packages/synapse-react-client/src/components/TimelinePlot/TimelinePhase.tsx

```diff
@@ -87,12 +104,17 @@ const getLayout = (
       height: 15,
     },
   ]
-  if (annotateTime && annotateTimeUnits) {
+  if (annotateTime && annotateTimeUnits && annotateEventCount) {
+    const eventCountText =
+      annotateEventCount > 1 ? `(${annotateEventCount})` : ''
     const x = start.add(annotateTime, annotateTimeUnits)
     annotations.push({
       x: x.format(),
       y: -1,
-      text: `${annotateTime} ${pluralize(annotateTimeUnits, annotateTime)}`,
+      text: `${annotateTime} ${pluralize(
+        annotateTimeUnits,
+        annotateTime,
+      )} ${eventCountText}`,
```

Comment:
I think `text` will have a trailing space when `eventCountText` is an empty string, e.g. `1 month ` -- just wondering if that was intentional, e.g. to adjust the spacing of the label to the axis tick mark?

Reply:
Noted!  The extra space will not shift the plot (since we are explicitly positioning the annotations), so I don't think it will have any impact on the result.  Thanks for pointing this out!